### ISSUE TITLE
long press on iPhone 6 doesn't bring up tool bar

### ISSUE THREAD

**Issue by emkayonline** [larki/VimIOS#1]
Your port of Vim to iOS is excellent, and works well on the iPad, but long pressing on the iPhone 6 screen flashes up the toolbar then it disappears again.

**Comment by larki** [comment]
Hi, I haven’t worked on the port in a while. I will try to fix the problem as soon as I can.

**Comment by emkayonline** [comment]
That is very cool of you, you have made the Applidium project into something great.

As a side note, do you know if it is possible to use the iOS clipboard in your port of Vim?  I’ve tried the registers \* and +, but no luck.

Thank you once again,

Martin

**Comment by larki** [comment]
No, clipboard should not work. I am not sure how much work it would be to implement this.

**Comment by emkayonline** [comment]
Thank you for saving my time.  That is good of you to let me know, you are very kind.

Martin
